snapcraft/cli/store.py

```diff
@@ -171,6 +172,24 @@ def push(snap_file, release):
     else:
         channel_list = None
 
+    # Review the snap.
```

Comment:
Might be best to pull this out into its own function, then it could be re-used for a possible `review` command future maybe?